src/Nancy/ErrorHandling/DefaultStatusCodeHandler.cs

```diff
@@ -62,47 +66,60 @@ public void Handle(HttpStatusCode statusCode, NancyContext context)
                 return;
             }
 
-            string errorPage;
-
-            if (!this.errorPages.TryGetValue(statusCode, out errorPage))
+            if (!this.errorMessages.ContainsKey(statusCode) || !this.errorPages.ContainsKey(statusCode))

```

Comment:
Yes, that's correct. This is a protection against calling this method without first checking `HandlesStatusCode`.